app/src/main/res/layout/comment_header.xml

```diff
@@ -0,0 +1,117 @@
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:adjustViewBounds="true"
+        android:scaleType="fitStart"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:src="@tools:sample/backgrounds/scenic" />
+
+    <View
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:background="?colorPrimary"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/commentHeaderTitle" />
+
+    <TextView
+        android:id="@+id/commentHeaderTitle"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingStart="10dp"
+        android:paddingTop="10dp"
+        android:textColor="@color/md_white_1000"
+        android:textSize="18sp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/commentHeaderImage"
+        tools:ignore="RtlSymmetry"
+        tools:text="Pretty Corn Field" />
+
+    <TextView
+        android:id="@+id/commentHeaderAuthorBy"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingStart="10dp"
+        android:text="by"
+        android:textColor="@color/md_white_1000"
+        app:layout_constraintBottom_toBottomOf="@id/commentHeaderAuthor"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/commentHeaderAuthor"
+        tools:ignore="RtlSymmetry" />
+
+    <TextView
+        android:id="@+id/commentHeaderAuthor"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingStart="4dp"
+        android:paddingTop="2dp"
+        android:paddingEnd="4dp"
+        android:paddingBottom="3dp"
+        android:textColor="@color/md_white_1000"
+        android:textStyle="bold"
+        app:layout_constraintStart_toEndOf="@id/commentHeaderAuthorBy"
+        app:layout_constraintTop_toBottomOf="@id/commentHeaderTitle"
+        tools:text="momodao" />
+
+    <TextView
+        android:id="@+id/commentHeaderDateTime"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:textColor="@color/md_white_1000"
+        app:layout_constraintBottom_toBottomOf="@id/commentHeaderAuthor"
+        app:layout_constraintStart_toEndOf="@id/commentHeaderAuthor"
+        app:layout_constraintTop_toTopOf="@id/commentHeaderAuthor"
+        tools:text=" 19 hours ago" />
+
+    <TextView
+        android:id="@+id/commentPoint"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:paddingStart="10dp"
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/worker8/Pixels/blob/4570d30e0729ae50f322f0ec7c6d3713c2b37a07/app/src/main/res/layout/comment_header.xml#L83"></span>When you define <code>paddingStart</code> you should probably also define <code>paddingEnd</code> for right-to-left symmetry</td>
    </tr>
  </tbody>
</table>